addons/medical_gui/initSettings.sqf

```diff
@@ -162,3 +162,12 @@ private _categoryColors = [ELSTRING(medical,Category), format ["| %1 |", LELSTRI
     [0, 5, 1],
     false
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(tourniquetWarning),
+    "CHECKBOX",
+    [LSTRING(TourniquetWarning_DisplayName), LSTRING(TourniquetWarning_Description)],
+    [ELSTRING(medical,Category), LSTRING(SubCategory)],
+    false,
+    false
```

Comment:
```suggestion
    true
```
Global

Reply:
Oh really? It seems like something that would be left to the player unless admins specifically care. That's my assumption for UI stuff in general.

Reply:
> Oh really? It seems like something that would be left to the player unless admins specifically care. That's my assumption for UI stuff in general.

I agree with this. @LinkIsGrim Why would you want to enforce it globally?

Reply:
Given the effect, I'd rather this be *really* deliberate on when it's enabled.

That being said, vox populi and all that.

Reply:
I don't think this needs to be global. Communits admins can force if they want to.